src/locales/strings/enUS.json

```diff
@@ -39,6 +39,14 @@
   "azteco_invalid_code": "Invalid Azteco code.",
   "azteco_service_unavailable": "Error contacting Azteco server. Please try again later.",
   "azteco_btc_only": "Please choose a BTC wallet and scan again.",
+  "bitwage_error_unsupported_chains": "Edge does not support the following chains listed in the request: %s",
```

Comment:
Remove reference to `bitwage` this is NOT a bitwage specific feature. It's the Request for Address protocol